src/x/xApplyFunctionStr.ts

```diff
@@ -30,14 +30,14 @@ export function xApplyFunctionStr(
   const fct = new Function(
     variableLabel,
     `return Number(${fctString
-      .replace(/(?<before>^|\W)(?<after>[a-z0-9]{2,}\()/g, '$1Math.$2')
+      .replace(/(?<before>^|\W)(?<after>[\da-z]{2,}\()/g, '$1Math.$2')
       .replace(/Math\.Math/g, 'Math')})`,
   );
   const toReturn = Float64Array.from(array);
   for (let i = 0; i < array.length; i++) {
     toReturn[i] = fct(array[i]);
-    if (isNaN(toReturn[i])) {
-      throw Error(
+    if (Number.isNaN(toReturn[i])) {
```

Comment:
@lpatiny Note that `Number.isNaN` and `isNaN` are not the same functions (in fact, this instance cannot be auto-fixed because of the different behavior).
It is generally wrong to use `isNaN` because it will first convert the value to a number before doing the check. For example, `isNaN('42') === true` but `Number.isNaN('42') === false`.

Reply:
when I try in Chrome console I obtain
- isNaN('42') -> false
- Number.isNaN('42') -> false

On the other hand
- Number.isNaN('42.123 AB') -> false
- isNaN('42.123 AB') -> true

isNaN is using Number() and Number.isNaN is using parseFloat() ???

Reply:
Sorry I mixed myself up in my example. The point is that `Number.isNaN` doesn't convert anything. It only returns true for the value `NaN`.

Reply:
So only `Number.isNaN(NaN)` this is true right ?

Reply:
Yes.